### PR TITLE
Add Tagline

### PR DESCRIPTION
**Pull request by AndrewDiMola**
image

### CODE REVIEW

**Comment by AndrewDiMola** [comment]
## Before
image

## After
image

**Review by Atry — APPROVED** [review]

**Comment by Atry** [comment]
Looks pretty good!

**Comment by AndrewDiMola** [comment]
Thanks, @Atry! Going to add some padding back on the 3-col section, then merge!

File: website/pages/en/index.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ $user_name = 'Fred';
 echo "<tt>Hello <strong>$user_name</tt></strong>";
 
 // XHP: Typechecked, well-formed, and secure
-$user_name = 'Andrew';
+$user_name = 'Fred';
```
Comment (lexidor) [inline website/pages/en/index.js:59]: > re-immortalize Fred

❤️

**Pull request merged**